modules/openy_gc_demo/migrations/migrate_plus.migration.virtual_y_node_blog.yml

```
@@ -93,6 +96,14 @@ process:
     - plugin: migration
       migration: virtual_y_media_image
       source: image
+  field_gc_video_category:
+    -
+      plugin: migration
```

```suggestion
      plugin: migration_lookup
```
see https://www.drupal.org/node/2861226